### DISCUSSION TITLE
Full RHEL 9 Support?

### DISCUSSION THREAD

**Discussion by EricSoares-UConn** [gbook/nidb#128] [Q&A] [answered]
I see release [NiDB v2023.5.988](https://github.com/gbook/nidb/releases/tag/v2023.5.988) announced  [new] .rpm for RHEL 9 compatible (Rocky 9, AlmaLinux 9, RHEL 9) and the RPM for RHEL 9 was included in the assets. I see the RPM again in release [NiDB v2023.10.1033](https://github.com/gbook/nidb/releases/tag/v2023.10.1033) but not in releases after.

Are there plans to consistently release RMP's for RHEL 9? I need to migrate a system from RHEL 7 to a new server and I would prefer RHEL 9 but I want to make sure it will be supported going forward. 

Appreciate the help!

**Comment by gbook** [comment, marked as answer]
We had limited (no) interest in RHEL9, so I stopped building those RPMs. But I can certainly start releasing them again. I will build one soon and post it here.

One thing to note is that if you upgrade to a new OS or a new server, you may want to follow the directions on migrating an existing installation to a new server: https://docs.neuroinfodb.org/nidb/getting-started/upgrade/migrate-existing-installation

Backing up the data is essential when upgrading!

**Reply by EricSoares-UConn** [reply]
Thank you! This is great! Our IT Org is really only deploying new servers on RHEL 9. We will be following the documentation for a migration. Really appreciate your efforts.

**Reply by gbook** [reply]
It's posted now.

**Reply by EricSoares-UConn** [reply]
Thank you!
